verify/verify_test.go

```diff
@@ -51,12 +59,12 @@ func (s ecdsaSigner) ContainsID(id string) bool {
 	return s.PublicData().ContainsID(id)
 }
 
-func (ecdsaSigner) Type() string {
-	return data.KeyTypeECDSA_SHA2_P256
+func (ecdsaSigner) MarshalSigner() (*data.PrivateKey, error) {
+	return nil, errors.New("not implemented")
```

Comment:
What is going on here? Totally not obvious.

Reply:
Ah, the verify package doesn't need to marshal/unmarshal signers for keys, so I left as unimplemented -- I'll clarify that this is for the test